Updating to 1.34.0 gets "500 internal Server Error" on admin ==> diagnostics

### Vaultwarden Support String

not possible because diganostics pacge can not be reached (description see below)

### Vaultwarden Build Version

1.34.0

### Deployment method

Official Container Image

### Custom deployment method

_No response_

### Reverse Proxy

yes

### Host/Server Operating System

NAS/SAN

### Operating System Version

Synology DSM 7.2.2-72806 Update 3

### Clients

Web Vault

### Client Version

_No response_

### Steps To Reproduce

description see below


### Expected Result

Open diagnostics page

### Actual Result

500 Internal Server Error

### Logs

```text
[2025-05-26 22:15:10.433][panic][ERROR] thread 'rocket-worker-thread' panicked at 'called `Result::unwrap()` on an `Err` value: Error("unexpected character '-' while parsing major version number")': src/api/admin.rs:701
   0: vaultwarden::init_logging::{{closure}}
   1: std::panicking::rust_panic_with_hook
   2: std::panicking::begin_panic_handler::{{closure}}
   3: std::sys::backtrace::__rust_end_short_backtrace
   4: __rustc::rust_begin_unwind
   5: core::panicking::panic_fmt
   6: core::result::unwrap_failed
   7: vaultwarden::api::admin::diagnostics::{{closure}}
   8: vaultwarden::api::admin::diagnostics::into_info::monomorphized_function::{{closure}}
   9: rocket::server::<impl rocket::rkt::Rocket<rocket::phase::Orbit>>::route::{{closure}}
  10: rocket::server::hyper_service_fn::{{closure}}::{{closure}}
  11: tokio::runtime::task::raw::poll
  12: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
  13: tokio::runtime::scheduler::multi_thread::worker::run
  14: tokio::runtime::task::raw::poll
  15: std::sys::backtrace::__rust_begin_short_backtrace
  16: core::ops::function::FnOnce::call_once{{vtable.shim}}
  17: std::sys::pal::unix::thread::Thread::new::thread_start

```

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/e8bfd4aa-6cd1-4194-a592-92daa6997189)

![Image](https://github.com/user-attachments/assets/0afc7b77-f50f-40cd-9e07-34ed5cb6375d)

### Additional Context

I just updated to version 1.34.0
Everything works fine as far as I can see except the diagnostics page in the admin panel. Anytime I try to go to the diagnostics page, I get "500 Internal Server Error" as seen on the attached picture "Server_Error.jpg"

I attached the the log of my Docker Vaultwarden instance ("Screenshot_Vaultwarden.jpg") but I´m not able to generate a support string

--- a/src/api/admin.rs
+++ b/src/api/admin.rs
@@ -698,10 +698,14 @@ async fn diagnostics(_token: AdminToken, ip_header: IpHeader, mut conn: DbConn) 
     let web_vault_version = get_web_vault_version();
 
     // Check if the running version is newer than the latest stable released version
-    let web_ver_match = semver::VersionReq::parse(&format!(">{latest_web_build}")).unwrap();
-    let web_vault_pre_release = web_ver_match.matches(
-        &semver::Version::parse(&web_vault_version).unwrap_or_else(|_| semver::Version::parse("2025.1.1").unwrap()),
-    );
+    let web_vault_pre_release = if let Ok(web_ver_match) = semver::VersionReq::parse(&format!(">{latest_web_build}")) {
+        web_ver_match.matches(
+            &semver::Version::parse(&web_vault_version).unwrap_or_else(|_| semver::Version::parse("2025.1.1").unwrap()),
+        )
+    } else {
+        error!("Unable to parse latest_web_build: '{latest_web_build}'");
+        false
+    };
 
     let diagnostics_json = json!({
         "dns_resolved": dns_resolved,

